iceoryx_posh/include/iceoryx_posh/internal/popo/ports/client_port_roudi.hpp

```diff
@@ -70,7 +70,7 @@ class ClientPortRouDi : public BasePort
     const MemberType_t* getMembers() const noexcept;
     MemberType_t* getMembers() noexcept;
 
-    void handleCaProProtocollViolation(iox::capro::CaproMessageType messageType) noexcept;
+    void handleCaProProtocolViolation(iox::capro::CaproMessageType messageType) noexcept;
```

Comment:
Please add a const to the argument and I think the method can be made const as well.